### PR TITLE
Fix typos in docs and sources

### PR DESCRIPTION
**Pull request by kinow**
Hello, I learned about ScriptEngine yesterday at a meeting in the BSC, and started reading the docs last night. I found a couple of typos, so ran the IDE linter over the docs & sources. Let me know if there's any edit in this pull request that is incorrect, and I will edit the commit correcting it.

Cheers,
-Bruno

### CODE REVIEW

**Comment by uwefladrich** [comment]
That's great, thanks a lot for contributing @kinow ! :+1: 

I tend to use language support when writing the documentation at least but it appears that I did an imperfect job :grin: 

I think I found just one place where the correction is wrong. Will comment once the checks are passed.

**Review by uwefladrich — CHANGES_REQUESTED** [review]
Great job, thanks!

File: docs/sphinx/scripts.rst [hunk 1]
```diff
@@ -395,7 +395,7 @@ basename
 
 
 dirname
-    Returns the directory part of a path (i.e. the parth with the base name
+    Returns the directory part of a path (i.e. the path with the base name
```
[the remark this answers is not in the document]
Comment (kinow) [reply docs/sphinx/scripts.rst:398]: Aaahh, good catch @uwefladrich . Let me correct it. Thanks!

[the remark this answers is not in the document]
Comment (kinow) [reply docs/sphinx/scripts.rst]: Done.

**Review by uwefladrich — APPROVED** [review]
Great, looks good!

**Pull request merged**